extensions/landing-zone/variables.tf

```diff
@@ -22,6 +22,48 @@ variable "operator_helm_release_namespace" {
   default     = "ibm-mq-operator"
 }
 
+variable "create_queue_manager" {
+  type        = bool
+  description = "Flag to create a Queue Manager for the IBM MQ operator. Default is true."
+  default     = true
+}
+
+variable "queue_manager_name" {
+  type        = string
+  description = "Name of the IBM MQ Queue Manager. Default to mq-qm."
+  default     = "mq-qm"
+}
+
+variable "create_ibm_mq_queue_manager_namespace" {
+  type        = bool
+  description = "Flag to create the namespace where to create the IBM MQ Queue Manager. Default to true."
+  default     = true
+}
+
+variable "ibm_mq_queue_manager_namespace" {
+  type        = string
+  description = "Namespace to install the IBM MQ Queue Manager. Default to mq-qm-ns."
+  default     = "mq-qm-ns"
+}
+
+variable "queue_manager_license" {
+  type        = string
+  description = "IBM MQ Queue Manager license. More info on IBM MQ Queue Manager licenses and its usage can be seen here: https://www.ibm.com/docs/en/ibm-mq/9.3?topic=mqibmcomv1beta1-licensing-reference."
+  default     = "L-AXAF-JLZ53A"
```

Comment:
In the next PR we should remove the default value from this variable and having the adopter to explicitly set it. For testing purposes it should be then added into the test code